ansible_collection/README.md

```diff
@@ -0,0 +1,14 @@
+# HPE Ansible Collections
+
+There are currently 1 HPE Collections
+* [Nimble](https://galaxy.ansible.com/hpe/nimble)
+
+## Requirements
+- ansible version >= 2.9
+
+## Installation
+### Nimble
+```bash
+ansible-galaxy collection install hpe.nimble
```

Comment:
Can you omit this for now and elaborate on how to do a local install from this repo?